add documentation for cluster tokens rest api (#2150)
buildkite/docs: pages/apis/rest_api/clusters.md

      "created_at": "2013-08-29T10:10:03.000Z"
    }
  }
]
```

Required scope: `read_clusters`

Success response: `200 OK`

### Get a token

```bash
curl "https://api.buildkite.com/v2/organizations/{org.slug}/clusters/{cluster.id}/tokens/{token.id}"
```

```json
{
  "id": "b6001416-0e1e-41c6-9dbe-3d96766f451a",
  "graphql_id": "Q2x1c3RlclRva2VuLS0tYjYwMDE0MTYtMGUxZS00MWM2LTlkYmUtM2Q5Njc2NmY0NTFh",
  "description": "Windows agents",
  "url": "http://api.buildkite.com/v2/organizations/test/clusters/e4f44564-d3ea-45eb-87c2-6506643b852a/tokens/b6001416-0e1e-41c6-9dbe-3d96766f451a",
  "cluster_url": "http://api.buildkite.com/v2/organizations/test/clusters/e4f44564-d3ea-45eb-87c2-6506643b852a",
  "created_at": "2023-05-26T04:21:41.350Z",
  "created_by": {
    "id": "3d3c3bf0-7d58-4afe-8fe7-b3017d5504de",
    "graphql_id": "VXNlci0tLTNkM2MzYmYwLTdkNTgtNGFmZS04ZmU3LWIzMDE3ZDU1MDRkZQo=",
    "name": "Sam Kim",
    "email": "sam@example.com",
    "avatar_url": "https://www.gravatar.com/avatar/example",
    "created_at": "2013-08-29T10:10:03.000Z"
  }
}
```

Required scope: `read_clusters`

Success response: `200 OK`

### Create a token

> > 📘 Token values

Reviewer: I think this heading could convey the intention of the note more clearly. To me, it looks like the note is about security, but specifically when someone can and can't see the value of a token. Perhaps:

```suggestion
> 📘 Token visibility
```

> > The token value is only be included in the response for the request to create it. To ensure the security of tokens, their value isn't included in subsequent responses.

Reviewer: I think we should lead with the security aspect. How about:

```suggestion
> To ensure the security of tokens, the value is only included in the response for the request to create the token. Subsequent responses do not contain the token value.
```
